src/main/java/core/basesyntax/db/FruitTransaction.java

```diff
@@ -0,0 +1,42 @@
+package core.basesyntax.db;
+
+public class FruitTransaction {
```

Comment:
The class name `FruitTransaction` is a bit misleading as it doesn't only represent a transaction but also the state of a particular fruit stock. Consider renaming it to better reflect its purpose, for example, `FruitStock` or `FruitRecord`.